src/main/java/org/beta/tchap/identite/matrix/rest/room/RoomService.java

```diff
@@ -0,0 +1,47 @@
+package org.beta.tchap.identite.matrix.rest.room;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+public class RoomService {
+    private final RoomClient roomClient;
+
+    public RoomService(RoomClient roomClient) {
+        this.roomClient = roomClient;
+    }
+
+    public DirectRoomsResource listDMRooms() {
+        Map<String, ArrayList<String>> rawResponse = roomClient.listDMRooms("@tchap-identite-tchap.beta.gouv.fr:i.tchap.gouv.fr");
```

Comment:
I guess those properties should be externalized

Reply:
Sure :)